frontend/src/components/Tag/Tag.tsx

```diff
@@ -12,5 +12,5 @@ export interface TagProps extends Omit<ChakraTagProps, 'colorScheme'> {
 }
 
 export const Tag = (props: TagProps): JSX.Element => {
-  return <ChakraTag {...props} />
+  return <ChakraTag textStyle="caption-1" {...props} />
```

Comment:
I think this should go into the theme file

Reply:
i think you reviweed an old commit @__@  this was changed [here
](https://github.com/opengovsg/FormSG/pull/2384/commits/cb111d26df467e7e19a92e1624a21d081314643b)